src/addon/cc-jenkins-info.js

```diff
@@ -0,0 +1,149 @@
+import '../atoms/cc-img.js';
+import '../atoms/cc-flex-gap.js';
+import '../molecules/cc-block-section.js';
+import '../molecules/cc-block.js';
+import '../molecules/cc-error.js';
+import { css, html, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { i18n } from '../lib/i18n.js';
+import { skeletonStyles } from '../styles/skeleton.js';
+import { ccLink, linkStyles } from '../templates/cc-link.js';
+
+const infoSvg = new URL('../assets/info.svg', import.meta.url).href;
+
+const JENKINS_LOGO_URL = 'https://static-assets.cellar.services.clever-cloud.com/logos/jenkins.svg';
+const JENKINS_DOCUMENTATION = 'https://www.clever-cloud.com/doc/addons/jenkins/';
+
+/**
+ * A component to display various informations (Documentation, access, updates, ...) for a Jenkins service.
+ *
+ * 🎨 default CSS display: `block`
```

Comment:
The CSS display now has a dedicated custom `@cssdisplay` JSDoc tag.

* Remove this line and add this line before the `@prop`:

```
 * @cssdisplay block
```

You can see an example in https://github.com/CleverCloud/clever-components/blob/master/docs/cc-example-component.js#L46